### PR TITLE
Created netstandard version

### PR DESCRIPTION
**Pull request by stevetemple**
Moving this to netstandard

Have made it so that you need to supply a ICurrentUrlService to set it up, this allows us to remove HttpContext from it.

### CODE REVIEW

File: Gibe.Pager/Extensions/UriExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    public static Uri ExtendQuery(this Uri uri, object values)
+    {
+      return ExtendQuery(uri, values.GetType().GetProperties().ToDictionary
+      (
+          propInfo => propInfo.Name,
+          propInfo => { var value = propInfo.GetValue(values, null); return value?.ToString(); }
+      ));
+    }
+  }
+}
```
Comment (stevetemple) [inline Gibe.Pager/Extensions/UriExtensions.cs:72]: Brought this in from Gibe.Tools, trying to remove dependencies on that one size fits all library

File: Gibe.Pager/Interfaces/ICurrentUrlService.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Gibe.Pager.Interfaces
+{
+	public interface ICurrentUrlService
+	{
+		string CurrentUrl();
+	}
```
Comment (stevetemple) [inline Gibe.Pager/Interfaces/ICurrentUrlService.cs:10]: Removing dependency on HttpContext

Comment (tristanjthompson) [reply Gibe.Pager/Interfaces/ICurrentUrlService.cs:10]: Would this need to be implemented by whatever is consuming the package?

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+`install-package Gibe.Pager`
+
+or for >= .NET 5.0 
+
+`dotnet add package Gibe.Pager`
+
+You'll need to init DI via
+
+`services.AddGibePager<T>()` where T is your `ICurrentUrlService` implementation
```
Comment (tristanjthompson) [inline readme.md]: Maybe make it a bit more explicit that you need to implement the `ICurrentUrlService`?
```suggestion
You'll need to implement the `ICurrentUrlService`.

Then, you'll need to init the DI via

`services.AddGibePager<T>()` where T is your `ICurrentUrlService` implementation
```

File: Gibe.Pager/Interfaces/ICurrentUrlService.cs (hunk 2 above)
Comment (tristanjthompson) [reply Gibe.Pager/Interfaces/ICurrentUrlService.cs:10]: Ignore me - I got to the readme!

**Review by tristanjthompson — APPROVED** [review]
Looks good - one thought on the readme, but nothing showstopping!

**Review by ThomasAdams2 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by lukehook — APPROVED** [review]

**Pull request merged**